Library/Homebrew/github_packages.rb

```diff
@@ -101,6 +101,18 @@ def self.root_url(org, repo, prefix = URL_PREFIX)
     "#{prefix}#{org}/#{repo_without_prefix(repo)}"
   end
 
+  def self.sanitize_root_url(url)
+    if url.to_s.start_with? "docker://"
```

Comment:
May as well do the match here (and same below) so you don’t end up with the prefix being present but the match failing.